src/App.css

```diff
@@ -1,3 +1,17 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
```

Comment:
- [ ] Make sure styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) and let's implement the changes by the previous reviewer. Kindly compare your design vs Zeplin design below 👇 
- Your design 👇 
![Screenshot (73)_LI](https://user-images.githubusercontent.com/67087664/146591319-a49c87a7-5b38-4ad3-bdd8-e5be30afc308.jpg)
- Zeplin design 👇 
![zeplin_LI](https://user-images.githubusercontent.com/67087664/146591374-2938d833-1031-40d0-a13d-99bb8173a38d.jpg)